plugins/discovery-azure-classic/src/main/java/org/elasticsearch/discovery/azure/classic/AzureUnicastHostsProvider.java

```diff
@@ -108,6 +109,7 @@ public static Deployment fromString(String string) {
     public AzureUnicastHostsProvider(Settings settings, AzureComputeService azureComputeService,
                                      TransportService transportService, NetworkService networkService) {
         super(settings);
```

Comment:
I'll drop this call and the ones like it in a follow up change, but it'll be quite large so I wanted to separate it from this one.